.github/workflows/prepare-transaction.yml

```
@@ -0,0 +1,207 @@
+name: "Validate the issue description and prepare the transaction"
+on:
+  issues:
+    types: [opened, edited]
+env:
+  ISSUE_BODY: ${{ github.event.issue.body }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  NUMBER: ${{ github.event.issue.number }}
+  REPO: ${{ github.repository }}
+  FACTORY_ADDRESS: "0x914d7Fec6aaC8cd542e72Bca78B30650d45643d7"
+  FACTORY_BYTECODE: "0x604580600e600039806000f350fe7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe03601600081602082378035828234f58015156039578182fd5b8082525050506014600cf3"
+  FACTORY_DEPLOYER_ADDRESS: "0xE1CB04A0fA36DdD16a06ea828007E35e1a3cBC37"
+  ERROR_MSG_RPC_FAILURE: >
+    **⛔️ Error:**<br>
+    Could not fetch the chain ID from the RPC URL.<br>
+    Either the RPC URL is missing, or the RPC is not responding.<br>
+    **Make sure your issue follows the issue template and includes the RPC URL.**<br>
+    :sparkles: You can edit the issue to trigger the check again. :sparkles:
+  ERROR_MSG_CHAINLIST_FAILURE: >
+    **⛔️ Error:**<br>
+    Couldn't find the chain in the chainlist. Please make sure it's added to the chainlist.<br>
+    For more information on how to add a chain, please refer to the [chainlist repository](https://github.com/ethereum-lists/chains).<br>
+    :sparkles: You can edit the issue to trigger the check again. :sparkles:
+  ERROR_MSG_PREFUND_CHECK: >
+    **⛔️ Error:**<br>
+    There was an error while estimating the deployment transaction and checking the pre-fund. Please make sure that the RPC URL is valid and reachable.<br>
+    :sparkles: You can edit the issue to trigger the check again. :sparkles:
+  SUCCESS_MSG: >
+    **✅ Success:**<br>
+    The issue description is valid:<br>
+    - The RPC URL is valid<br>
+    - The chain is in the chainlist<br>
+    - The deployer address is pre-funded<br>
+    :sparkles: The team will be in touch with you soon :sparkles:
+
+jobs:
+  prepareTransaction:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate the issue description and get the RPC URL
+        id: validate_description
+        run: |
+          trim() {
+              # 3.5.3 Shell Parameter Expansion at https://www.gnu.org/software/bash/manual/bash.html
+              local var="$*"
+              # remove leading whitespace characters
+              var="${var#"${var%%[![:space:]]*}"}"
+              # remove trailing whitespace characters
+              var="${var%"${var##*[![:space:]]}"}"
+              echo "$var"
+          }
+
+          rpc_url=$(echo $ISSUE_BODY | egrep -o 'https?://[^ ]+' -m 1 | head -1)
+
+          if [ -z "$rpc_url" ]; then
+            echo "COMMENT_OUTPUT="$ERROR_MSG_RPC_FAILURE"" >> $GITHUB_ENV
+            exit 1
+          fi
+
+          rpc_url=$(trim $rpc_url)
```

I would "quote" this:
```suggestion
          rpc_url="$(trim "$rpc_url")"
```